test/replcompletions.jl

```diff
@@ -494,6 +493,30 @@ c, r, res = test_scomplete(s)
         ENV["PATH"] = oldpath
     end
 
+    # Make sure completion results are unique in case things are in the env path twice.
+    let
+        # Create /tmp/repl-completion and /tmp/repl-completion-subdir/repl-completion
+        file0 = joinpath(tempdir(), "repl-completion")
+        dir = joinpath(tempdir(), "repl-completion-subdir")
+        mkdir(dir)
+        file1 = joinpath(dir, "repl-completion")
+        touch(file0)
+        touch(file1)
+
+        withenv(
+            function()
+                s = string("repl-completio")
```

Comment:
Oh, nice.
Changed to use the new syntax (I'll do the other tests in another PR).